test/e2e/feature/feature.go

```diff
@@ -190,19 +193,26 @@ var (
 	// TODO: document the feature (owning SIG, when to use this feature for a test)
 	NEG = framework.WithFeature(framework.ValidFeatures.Add("NEG"))
 
-	// TODO: document the feature (owning SIG, when to use this feature for a test)
+	// Owner: sig-network
+	// Marks tests that require working external DNS.
 	NetworkingDNS = framework.WithFeature(framework.ValidFeatures.Add("Networking-DNS"))
 
-	// TODO: document the feature (owning SIG, when to use this feature for a test)
+	// Owner: sig-network
+	// Marks tests that require connectivity to the Internet via IPv4
 	NetworkingIPv4 = framework.WithFeature(framework.ValidFeatures.Add("Networking-IPv4"))
 
-	// TODO: document the feature (owning SIG, when to use this feature for a test)
+	// Owner: sig-network
+	// Marks tests that require connectivity to the Internet via IPv6
 	NetworkingIPv6 = framework.WithFeature(framework.ValidFeatures.Add("Networking-IPv6"))
 
-	// TODO: document the feature (owning SIG, when to use this feature for a test)
+	// Owner: sig-network
+	// Marks a single test that creates potentially-disruptive amounts of network
+	// traffic between nodes.
```

Comment:
Should we extend the definition of `Feature:` to cover this case?

This looks like it was chosen because of the default skip for anything that starts with `Feature:`, which kind of makes sense - it's just a bit odd.

Reply:
Yeah, I don't know if we _should_ be doing this, but we definitely _are_ doing this, so I documented it...

Reply:
Yes, makes sense.

In https://github.com/kubernetes/enhancements/pull/3042#discussion_r989232432 and elswhere around that KEP we were discussing naming of `Feature:` vs. `FeatureGate:`.

In the light of this particular usage here I believe `Special:` might be a better name: "this test needs special cluster configurations to pass and/or exhibits special behavior that makes it unsuitable for normal testing". However, I also believe that renaming it is not worth the churn.

Instead, I'll add some additional comments to https://github.com/kubernetes/community/pull/7824

/cc @aojea

Reply:
yeah, this test is a bit hard to qualify as e2e and here, it is also duplicated in https://github.com/kubernetes/perf-tests , that is a best place for these kind of tests ... but I remember at RedHat some people used it so maybe keeping as is is ok